scene/gui/text_edit.cpp

```diff
@@ -6990,6 +6994,14 @@ void TextEdit::_reset_caret_blink_timer() {
 	}
 }
 
+bool TextEdit::is_caret_on_readonly() {
+	return caret_on_readonly;
+}
+
+void TextEdit::set_caret_on_readonly(bool p_value) {
+	caret_on_readonly = p_value;
+}
+
```

Comment:
This should follow the header, so should be placed under `get_caret_blink_interval`